src/main/java/io/supertokens/inmemorydb/queries/GeneralQueries.java

```diff
@@ -186,6 +186,23 @@ public static void createTablesIfNotExists(Start start, Main main) throws SQLExc
             update(start, UserIdMappingQueries.getQueryToCreateUserIdMappingTable(start), NO_OP_SETTER);
         }
 
+        if (!doesTableExists(start, Config.getConfig(start).getTotpUsersTable())) {
+            getInstance(main).addState(CREATING_NEW_TABLE, null);
+            update(start, TOTPQueries.getQueryToCreateUsersTable(start), NO_OP_SETTER);
+        }
+
+        if (!doesTableExists(start, Config.getConfig(start).getTotpUserDevicesTable())) {
+            getInstance(main).addState(CREATING_NEW_TABLE, null);
+            update(start, TOTPQueries.getQueryToCreateUserDevicesTable(start), NO_OP_SETTER);
```

Comment:
Should we add some indices here?

Reply:
Depends on the queries being used. Are there any queries for this table that don't depend on the primary key?

Reply:
@KShivendu - you didn't reply to this